controllers/users_controller.js

```diff
@@ -1,5 +1,22 @@
+
 module.exports.findAllUsers = (req, res) => {
-    res.status(202).json({data : "success"})
+	const cassandra = require('cassandra-driver');
+	const client = new cassandra.Client({
+		contactPoints: ['127.0.0.1']
+	});
+
+	client.connect()
+	  .then(function () {
+	    console.log('Connected to cluster with %d host(s): %j', client.hosts.length, client.hosts.keys());
+	    console.log('Keyspaces: %j', Object.keys(client.metadata.keyspaces));
+	    console.log('Shutting down');
+	    return client.shutdown();
+	  })
+	  .catch(function (err) {
+	    console.error('There was an error when connecting', err);
+	    return client.shutdown();
+	  });
```

Comment:
You should move the connection code to `index.js` or somewhere else that isn't a request

Reply:
true, will change